docs/src/pages/components/rating/index.page.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Rating
+description: The rating component displays a read only star rating
```

Comment:
Perhaps rephrase to: "The Rating component displays a read-only star rating of a product or service."